### ISSUE TITLE
Query about whitelist change

### ISSUE THREAD

**Issue by potomato** [itzg/docker-minecraft-bedrock-server#103]
Hi,

I used to set WHITE_LIST="true" and map a whitelist file from host to /data/whitelist.json, which worked fine.
`./survival/whitelist.json:/data/whitelist.json`

From today that fails with `Device or resource busy` when trying to delete whitelist.json.

It seems now that after #101 that WHITE_LIST is now comma-separated list of usernames for [bedrock-entry.sh](https://github.com/itzg/docker-minecraft-bedrock-server/blob/77f52927b88e840a2fb924b4b6cba128f46f104d/bedrock-entry.sh#L100)

That's all fine and I've switched over to the new approach but the README has:

> -e WHITELIST "player1,player2,player3"

This should be WHITE_LIST shouldn't it?

Happy to do a PR for the _ if you like!

Thanks.

**Comment by itzg** [comment]
Oh yeah, I missed that during the review.

@hyperbolic2346 , perhaps you could fix that up while you resolve the merge conflicts in #102 ?

**Comment by itzg** [comment]
...and was just thinking, maybe that variable should be called something distinct like `WHITE_LIST_USERS` to avoid any other configuration impact like you encountered @potomato .

What do you think @hyperbolic2346 ? Unfortunately that would mean one more small PR to the helm chart.

**Comment by potomato** [comment]
Might be a good idea to use WHITE_LIST_USERS for this. At the moment it might be hard for new users to determine when it is list of names vs when it's boolean.

Is there any way of supporting both ways - so we could still use an external whitelist.json? Happy to help out a bit if that helps.

**Comment by hyperbolic2346** [comment]
Yeah, `WHITE_LIST_USERS` would allow setting users into the whitelist and it would imply `WHITE_LIST=true` as well. Then you can `WHITE_LIST=true` if you wanted to "power user" your way into some custom `whitelist.json` file. I'm fine with that and can make that change soon. Probably won't piggyback on #102 though as it seems like a new PR to me.
